config/routes.rb

```diff
@@ -3,10 +3,7 @@
 
   namespace :api do
     namespace :v1 do
-      namespace :admin do
-        resources :mentors, only: [:create, :update, :destroy]
```

Comment:
We should keep the DESTROY in case any mentors are created by accident, etc., to cover all bases.

Reply:
Added this to a new branch.

Reply:
awesome